### ISSUE TITLE
watson edit: add --from, --to switches

### ISSUE THREAD

**Issue by abarger-bss** [TailorDev/Watson#309]
One of the most common things I do in Watson is edit my current frame because I started or stopped watson well after I started or stopped my work. It would be nice if I could edit the dates for a frame directly in the CLI with --from or --to switches on the edit command.

**Comment by jmaupetit** [comment]
Thanks for the report @abarger-bss! Do you have time to implement such feature and propose a PR? :pray:

**Comment by benallan** [comment]
Hi, I actually was hoping to use these same options too so I opened a PR for them: #333. I hope you don't mind, @abarger-bss.

**Comment by abarger-bss** [comment]
Not at all, I'm relieved. Sorry for ghosting on this issue - Python's not my daily tool so I would have had a hard time with this. Glad to hear the suggestion makes sense for others though.

I will add that since opening this issue I've had good results using `watson stop --at <time>` and `watson start --no-gap` to set frame times without a text editor.

**Comment by benallan** [comment]
Thanks a lot for reminding me to check the `stop --at` option, I really needed to see which time formats it accepts. I also like using `stop --at`, but I often seem to be so forgetful that I need to edit the times for 2 or 3 frames back... So I suppose that new start/stop options for the edit command might still be useful.
